manifests/agent.pp

```diff
@@ -27,6 +27,13 @@
 # [*manage_repo*]
 #   When true, it will create repository for installing the agent.
 #
+# [*manage_choco*]
+#   When true on windows, it will use chocolatey to install the agent.
```

Comment:
Can you update this text? We should mention that https://forge.puppet.com/puppetlabs/chocolatey is required for this option